tests/index.jsx

```diff
@@ -0,0 +1,25 @@
+import Test from '../src/legit-tests'
+import { Find } from '../src/middleware'
+import { expect } from 'chai';
+
+import TableComponent from './components/table-component'
+
+describe('Index middleware', () => {
+
+  it('should select a specific element after a multi find', () => {
+
+    Test(<TableComponent/>)
+      .find('table', {multi: false})
+      .find('td', {multi: true})
+      .index(1)
+      .element(td => {
+        expect(td.getDOMNode().textContent).to.equal('2')
+      })
+  });
+
+  it('should throw an error if the root is not multi', () => {
+
+    var search = () => Test(<TableComponent/>).find('table', {multi: false}).index(0)
```

Comment:
@geowarin I think you missed this. Also Change the file name from `index.jsx` to something else. It is slightly misleading at first site becaues index.js is used to export stuff but this concers with index middleware.